docs/source/kinds_of_types.rst

```diff
@@ -739,7 +742,9 @@ Example:
     packet = PacketId(100, 100)
     tcp_packet = TcpPacketId(packet)  # OK
 
-    tcp_packet = TcpPacketId(127, 0)  # Fails in type checker and at runtime
+    tcp_packet = TcpPacketId(127, 0)  # error: Too many arguments for "TcpPacketId"
```

Comment:
Perhaps, it would be good to point out that this line fails both in typechecker and at runtime.